src/AdoNet/Orleans.Persistence.AdoNet/Orleans.Persistence.AdoNet.csproj

```diff
@@ -0,0 +1,87 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Persistence.AdoNet</PackageId>
+    <Title>Microsoft Orleans Persistence AdoNet Providers</Title>
+    <Description>Microsoft Orleans persistence providers for AdoNet Providers</Description>
```

Comment:
Should these "AdoNet"s be _ADO.NET_ here? Are these used as search words in Nuget? ADO.NET is the official term, I think.